components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/gen/java/org/wso2/carbon/identity/api/server/application/management/v1/ApplicationsApi.java

```diff
@@ -429,6 +429,30 @@ public Response getApplication(@ApiParam(value = "ID of the application.",requir
         return delegate.getApplication(applicationId );
     }
 
+    @Valid
+    @GET
+    @Path("/{applicationId}/authenticators/{tenantDomain}")
```

Comment:
Why did you append {tenantDomain} at the end?
Since the API need to be invoked in /t/{tenantDomain}/.. path do we need to have the additional {tenantDomain} path param at the end?

Reply:
This API will be invoked from application signup page. api will be invoked without tenanted path.
We can invoke this API without passing the tenant-domain as well.